.eslintrc.json

```diff
@@ -1,3 +1,4 @@
 {
-  "extends": "tamia"
+	"extends": "tamia",
```

Comment:
```suggestion
	"extends": "tamia/react",
```

And you don't need to change the parser ;-)

Reply:
Thanks, will do.